src/partitioned-dbs/index.rst

```diff
@@ -0,0 +1,384 @@
+        "field_name": "Bob's Corn Field #5",
+        "readings": [
+            ["2019-01-21T00:00:00", 0.15],
+            ["2019-01-21T06:00:00", 0.14],
+            ["2019-01-21T12:00:00", 0.16],
+            ["2019-01-21T18:00:00", 0.11]
+        ]
+    }
+    shell> $ curl -X POST -H "Content-Type: application/json" \
+                http://127.0.0.1:5984/my_new_db -d @doc.json
+    {
+        "ok": true,
+        "id": "sensor-260:sensor-reading-ca33c748-2d2c-4ed1-8abf-1bca4d9d03cf",
+        "rev": "1-05ed6f7abf84250e213fcb847387f6f5"
+    }
+
+The only change required to the first example document is that we are now
+including the partition name in the document id by prepending it to the
+old id separated by a colon.
+
+.. note::
+
+    The partition name in the document id is not magical. Internally,
+    the database is simply using only the partition for hashing
+    the document to a given shard, instead of the entire document id.
+
+Working with documents in a partitioned database is no different than
+a non-partitioned database. All APIs are available, and existing client
+code will all work seamlessly.
+
+Now that we have created a document, we can get some info about the partition
+containing the document:
+
+.. code-block:: bash
+
+    shell> curl http://127.0.0.1:5984/my_new_db/_partition/sensor-260
+    {
+      "db_name": "my_new_db",
+      "doc_count": 1,
+      "doc_del_count": 0,
+      "partition": "sensor-260",
+      "sizes": {
+        "active": 244,
+        "external": 347
+      }
+    }
+
+And we can also list all documents in a partition:
+
+.. code-block:: bash
+
+    shell> curl http://127.0.0.1:5984/my_new_db/_partition/sensor-260/_all_docs
+    {"total_rows": 1, "offset": 0, "rows":[
+        {
+            "id":"sensor-260:sensor-reading-ca33c748-2d2c-4ed1-8abf-1bca4d9d03cf",
+            "key":"sensor-260:sensor-reading-ca33c748-2d2c-4ed1-8abf-1bca4d9d03cf",
+            "value": {"rev": "1-05ed6f7abf84250e213fcb847387f6f5"}
+        }
+    ]}
+
+Note that we can use all of the normal bells and whistles available to
+``_all_docs`` requests. Accessing ``_all_docs`` through the
+``/dbname/_partition/name/_all_docs`` endpoint is mostly a convenience
+so that requests are guaranteed to be scoped to a given partition. Users
+are free to use the normal ``/dbname/_all_docs`` to read documents from
+multiple partitions.
+
+Next, we'll create a design document containing our index for
+getting all readings from a given sensor. The map function is similar to
+our earlier example except we've accounted for the change in the document
+id.
+
+.. code-block:: javascript
+
+    function(doc) {
+        if(doc._id.indexOf(":sensor-reading-") < 0) {
+            return;
+        }
+        for(var r in doc.readings) {
+            emit([doc.sensor_id, r[0]], r[1])
+        }
+    }
+
+We can upload our design document and try out a partitioned
+query:
+
+.. code-block:: bash
+
+    shell> cat ddoc.json
```

Comment:
Since you leave out the PUT of the ddoc, maybe just skip to the `curl` statement at L297 here. I get you're trying to show the lack of the `partitioned: false` value, so the other option is to show an ellipsis (`...`) between this and the next `curl` to show you're skipping the ddoc upload in the example.

Reply:
Addressed in 347a6ce.